uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/core/AboutThisApp.kt

```diff
@@ -0,0 +1,51 @@
+package io.github.droidkaigi.feeder.core
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.feeder.core.theme.typography
+
+@Preview
+@Composable
+fun AboutThisApp() {
+    Column(
+        modifier = Modifier
+            .padding(vertical = 48.dp, horizontal = 20.dp)
+    ) {
+        AboutThisAppComponent()
+        Spacer(modifier = Modifier.height(68.dp))
+        WhatIsDroidKaigiComponent()
+    }
+}
+
+@Composable
+fun AboutThisAppComponent() {
+    Text(
```

Comment:
I think wrap by Column() is more better.
Because if AboutThisAppComponent() called without Column(), all texts stacked.

```suggestion
    Column() {
        Text()
        Spacer()
        Text()
    }
```

Please refer below:
https://developer.android.com/jetpack/compose/layout

![image](https://user-images.githubusercontent.com/5106629/109416628-c6dfa180-7a02-11eb-9c85-598026d208a8.png)

Reply:
@tomoya0x00 Thanks for the kind review and the code suggestion. d5a73eb